### PR TITLE
Setting up Bottom Navigation

### PR DESCRIPTION
**Pull request by cristhianescobar**
Setting up bottom bar navigation based on current design

### CODE REVIEW

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            onClick = { onNavigationSelected(Screen.HomeRoot) }
+        )
+        HomeBottomNavigationItem(
+            icon = R.drawable.ic_dashboard_black_24dp,
+            selected = currentRootScreen == Screen.TribeRoot,
+            label = stringResource(R.string.tribe),
+            onClick = { onNavigationSelected(Screen.TribeRoot) }
+        )
+        HomeBottomNavigationItem(
+            icon = R.drawable.ic_notifications_black_24dp,
```
Comment (andymiller813) [inline androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt]: we should be able to get the proper icons now but we can do that as a follow up

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.projectnewm.R
+import kotlinx.coroutines.flow.collect
+
+@Composable
+internal fun NEWMApp(navController: NavHostController = rememberNavController()) {
+    Scaffold(
+        bottomBar = {
+            val currentRootScreen by navController.currentRootScreenAsState()
```
Comment (andymiller813) [inline androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt:19]: Is there any way to hoist this state up a level so it gets remembered and we don't have to go find the entry tab each time?

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.navigation.NavController
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.projectnewm.R
+import kotlinx.coroutines.flow.collect
+
+@Composable
+internal fun NEWMApp(navController: NavHostController = rememberNavController()) {
```
Comment (andymiller813) [inline androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt]: nit: 
```suggestion
internal fun NewmApp(navController: NavHostController = rememberNavController()) {
```

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            NEWMBottomNavigation(
+                currentRootScreen = currentRootScreen,
+                onNavigationSelected = { navController.navigate(it.route) }
+            )
+        }
+    ) { }
+}
+
+@Composable
+internal fun NEWMBottomNavigation(
```
Comment (andymiller813) [inline androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt]: nit: 
```suggestion
internal fun NewmBottomNavigation(
```

**Review by andymiller813 — APPROVED** [review]

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt (hunk 1 above)
Comment (cristhianescobar) [reply androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt]: I added the icons on the last commit, however they seem to need a bit of work from design, they don't scale well on the app on their current form

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt (hunk 2 above)
Comment (cristhianescobar) [reply androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt:19]: We can, the next PR that sets up the nav flow will have it, good call out

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt (hunk 3 above)
Comment (cristhianescobar) [reply androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt]: Change made 👍🏼

File: androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt (hunk 4 above)
Comment (cristhianescobar) [reply androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt]: Change made 👍🏼

**Pull request merged**